TomEE-7.0/jre8/Temurin/ubuntu/plume/Dockerfile

```diff
@@ -1,10 +1,14 @@
-FROM openjdk:8-jre-bullseye
+FROM eclipse-temurin:8-jre-focal
```

Comment:
Would we be better moving to 8-jre-jammy?

Reply:
If we move to Jammy, I suggest moving to Jammy for TomEE 8.0, too, so they are all equal.

I’ve chosen Focal, because it is the closest match to the previous Debian Bullseye basis. Jammy is Bookworm based.

Reply:
Good point.  I'll merge it in the morning.  I can have a discussion with the developers and get more input.